etc/config.sample.toml

```diff
@@ -255,6 +255,12 @@
   # the request log to stderr.
   # access-log-path = ""
 
+  # Filters which requests should be logged. Each filter is of the pattern NNN, NNX, or NXX where N is
+  # a number and X is a wildcard for any number. To filter all 5xx responses, use the string 5xx.
```

Comment:
`a number (1-5, inclusive)` perhaps?

Reply:
I really like the string pattern because it feels natural to me.